discord/enums.py

```diff
@@ -218,6 +218,11 @@ def __str__(self) -> str:
         return self.name
 
 
+class MessageReferenceType(Enum):
+    default = 0
```

Comment:
This should be called `reply` with an alias to `default`. The API docs even call it `REPLY` and `DEFAULT` despite not mentioning the former. https://discord.com/developers/docs/resources/message#message-reference-types